### PR TITLE
Updated netmanager readme

### PR DESCRIPTION
**Pull request by Codebmk**
#### Summary of Changes (What does this PR do?)
- Updated Netmanager codebase README to guide new users in setting up and running the application locally

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [ ] I've tested this locally
- [x] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Go to files changed. Select view file

#### What are the relevant tickets?
- [ticket_id]()

### CODE REVIEW

**Review by 123MwanjeMike — APPROVED** [review]

**Review by NoahNsimbe — APPROVED** [review]
LGTM

File: netmanager/README.md [hunk 1]
```diff
@@ -1,68 +1,24 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Netmanager Application
+This application is used for monitoring, device registration, location registration. Can be accessed here: https://staging-platform.airqo.net/ and https://platform.airqo.net/
 
-## Available Scripts
+## Local Setup
+npm is our package manager of choice here. Check out setup instructions [here](https://nodejs.org/dist/) if you don't have it installed already. <br/>After that you can run `npm install` to install all the needed dependencies.<br/>
```
Comment (ja-odur) [inline netmanager/README.md:5]: @Codebmk You may want to put  the package installation (npm install) as its own section/sub-section. Also include the step of setting up the `.env` file (include the link to the google drive file).

**Review by ja-odur — CHANGES_REQUESTED** [review]

**Comment by ja-odur** [comment]
@Codebmk  I realised you based this branch off the [cross-env branch](https://github.com/airqo-platform/AirQo-frontend/pull/495) branch. This might affect the order in which it will be merged

File: netmanager/README.md (hunk 1 above)
Comment (Codebmk) [reply netmanager/README.md:5]: On it @ja-odur

**Review by FaithDaka — APPROVED** [review]

**Review by ja-odur — APPROVED** [review]
LGTM

**Pull request merged**